deploy/charts/jetstack-agent/README.md

```diff
@@ -2,136 +2,151 @@
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| affinity | object | `{}` |  |
-| authentication.createSecret | bool | `false` |  |
-| authentication.secretKey | string | `"credentials.json"` |  |
-| authentication.secretName | string | `"agent-credentials"` |  |
-| authentication.secretValue | string | `""` |  |
-| authentication.type | string | `"file"` |  |
-| config.cluster | string | `""` |  |
-| config.dataGatherers.custom | list | `[]` |  |
-| config.dataGatherers.default | bool | `true` |  |
-| config.organisation | string | `""` |  |
-| config.period | string | `"0h1m0s"` |  |
-| config.server | string | `"https://platform.jetstack.io"` |  |
-| fullnameOverride | string | `""` | Helm default setting, use this to shorten install name |
-| image | object | `{"pullPolicy":"IfNotPresent","repository":"quay.io/jetstack/preflight","tag":"v0.1.38"}` | image settings |
-| imagePullSecrets | list | `[]` | specify credentials if pulling from a customer registry |
-| nameOverride | string | `""` | Helm default setting to override release name, leave blank |
-| nodeSelector | object | `{}` |  |
-| podAnnotations | object | `{}` |  |
-| podSecurityContext | object | `{}` |  |
-| replicaCount | int | `1` | default replicas, do not scale up |
-| resources.limits.cpu | string | `"500m"` |  |
-| resources.limits.memory | string | `"500Mi"` |  |
-| resources.requests.cpu | string | `"200m"` |  |
-| resources.requests.memory | string | `"200Mi"` |  |
-| securityContext.capabilities.drop[0] | string | `"ALL"` |  |
-| securityContext.readOnlyRootFilesystem | bool | `true` |  |
-| securityContext.runAsNonRoot | bool | `true` |  |
-| securityContext.runAsUser | int | `1000` |  |
-| serviceAccount.annotations | object | `{}` | Annotations to add to the service account |
-| serviceAccount.create | bool | `true` | Specifies whether a service account should be created @default true |
-| serviceAccount.name | string | `""` |  |
-| tolerations | list | `[]` |  |
+Set this environment variable to contain the encoded agent credential:
 
-----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.10.0](https://github.com/norwoodj/helm-docs/releases/v1.10.0)
-# jetstack-agent
+```shell
+export HELM_SECRET="$(cat credentials.json | base64 -w0)"
+```
 
-![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.38.0](https://img.shields.io/badge/AppVersion-v1.38.0-informational?style=flat-square)
+Installing the chart with additional configuration options for the agents
+credential, read from the environment variable just set:
 
-Jetstack Secure Agent
+```shell
+helm upgrade --install --create-namespace -n jetstack-secure jetstack-agent \
+  oci://eu.gcr.io/jetstack-secure-enterprise/charts/jetstack-agent \
```

Comment:
recommendation - can we turn the registry path to a variable as most often this will be installed via a mirrored location